### ISSUE TITLE
Karfell Kennel-Master has English text for all localised prints

### ISSUE THREAD

**Issue by floifweb** [scryfall/gatherer-bugs#97]
[Karfell Kennel-Master](https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=504613&printed=true) from Kaldheim has all English text for its French translation.

Reported to Wizards.

**Comment by spacemonaut** [comment]
This is true for all languages; every localisation simply has the English text again for its localised print text:

* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=506720
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=503710
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=504613
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=504914
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=505215
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=505516
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=505817
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=506118
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=506419
* https://gatherer.wizards.com/Pages/Card/Details.aspx?printed=true&multiverseid=504312

**Comment by RivalRowan** [comment]
This issue is still present in Gatherer '25 as of September 3, 2025.

German - https://gatherer.wizards.com/KHM/de-de/101/karfell-kennel-master
Spanish - https://gatherer.wizards.com/KHM/es-es/101/karfell-kennel-master
French - https://gatherer.wizards.com/KHM/fr-fr/101/karfell-kennel-master
Italian - https://gatherer.wizards.com/KHM/it-it/101/karfell-kennel-master
Japanese - https://gatherer.wizards.com/KHM/ja-jp/101/karfell-kennel-master
Korean - https://gatherer.wizards.com/KHM/ko-ko/101/karfell-kennel-master
Portuguese - https://gatherer.wizards.com/KHM/pt-br/101/karfell-kennel-master
Russian - https://gatherer.wizards.com/KHM/ru-ru/101/karfell-kennel-master
Simplifiec Chinese - https://gatherer.wizards.com/KHM/zh-cn/101/karfell-kennel-master
Traditional Chinese - https://gatherer.wizards.com/KHM/zh-tw/101/karfell-kennel-master

<img width="1297" height="710" alt="Portuguese-language Karfell Kennel-Master (Kaldheim)" src="https://github.com/user-attachments/assets/f456944f-29be-45fc-aa22-2f20761ac8b5" />
